### PR TITLE
UXUI-250 - adding a troubleshooting markdown

### PR DESCRIPTION
**Pull request by jodiedoubleday**
#### What does this PR do?
Create a TROUBLESHOOTING.md file to report any general and user specific issues when using the toolkit

#### What unit or integration tests does this PR have?
None

#### What selenium tests does this PR have?
None

#### How should a developer review this?
- Take a look at the file, have a good read
- I'm open to suggestions to how to improve the doc but not creating lists of other issues. Issues should be created on GitHub, this file offers only issues and solutions.

#### How should this be manually tested?
As above

#### Any background context you want to provide?
This is designed to stop common questions or issues being raised with the developers who created the toolkit.

#### What are the relevant tickets?
https://holidayextras.jira.com/browse/UXUI-250

#### What gif best describes how you feel about this work?
School bus

---
#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):
- [x] I've updated documentation with any changes to procedures.
- [x] I've checked this work against the requirements of the Jira.
- [x] I am ready for this to be code reviewed, merged and tested.
- [x] I've thought about writing a blog post about this.

#### IP or Developer Review:
- [x] I agree with the assumptions made in the quality checklist.
- [x] I’ve witnessed the work behaving as expected.
- [x] +1 from me!

#### Software Engineer or Developer review:
- [x] I agree with the assumptions made in the quality checklist.
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve checked for coding anti-patterns.
- [x] I've checked this work against the requirements of the Jira.
- [x] +1 from me!

#### Software Engineer or project guru review:
- [x] I agree with the assumptions made in the quality checklist.
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve checked for coding anti-patterns.
- [x] I don’t believe this work introduces unnecessary technical debt.
- [x] +1 from me!

### CODE REVIEW

**Comment by stuartgiles** [comment]
Nice! Could you also add a link to this in the README.md for more info?

**Comment by Joezo** [comment]
I like this, we should do more of these :) As Stu said can you add a link to README?

**Comment by jodiedoubleday** [comment]
Added :) @stuartgiles @Joezo

**Comment by stuartgiles** [comment]
:+1:

**Comment by Joezo** [comment]
:+1: :bug:

File: README.md [hunk 1]
```diff
@@ -45,6 +45,13 @@ cd ui-toolkit
 npm install
 ```
 
+Troubleshooting
+---
+If you have any problems installing the ui-toolkit then please view the
+[troubleshooting guide](https://github.com/holidayextras/ui-toolkit/TROUBLESHOOTING.md) which contains common errors and
+solutions. If your cannot find your specific problem then please create a
```
Comment (shackpank) [inline README.md]: should be "if you cannot find your"

**Comment by shackpank** [comment]
Typo, otherwise all good +1

**Pull request merged**